tests/unit/test_utils.py

```diff
@@ -2,19 +2,21 @@
 # See LICENSE file for licensing details.
 
 import re
-import unittest
+from unittest import TestCase
 
 from utils import new_password
 
+# used for assert functions
+tc = TestCase()
 
-class TestUtils(unittest.TestCase):
-    def test_new_password(self):
-        # Test the password generation twice in order to check if we get different passwords and
-        # that they meet the required criteria.
-        first_password = new_password()
-        self.assertEqual(len(first_password), 16)
-        self.assertIsNotNone(re.fullmatch("[a-zA-Z0-9\b]{16}$", first_password))
 
-        second_password = new_password()
-        self.assertIsNotNone(re.fullmatch("[a-zA-Z0-9\b]{16}$", second_password))
-        self.assertNotEqual(second_password, first_password)
+def test_new_password():
+    # Test the password generation twice in order to check if we get different passwords and
+    # that they meet the required criteria.
+    first_password = new_password()
+    tc.assertEqual(len(first_password), 16)
+    tc.assertIsNotNone(re.fullmatch("[a-zA-Z0-9\b]{16}$", first_password))
```

Comment:
We can just use regular `assert`s in such cases. No need to keep the test case just for that.

Reply:
agreed, will refactor this together with merging the 2 suites in the follow-up PR.